docs/verticalpodautoscaler-metrics.md

```diff
@@ -11,6 +11,97 @@
 | kube_verticalpodautoscaler_labels                                          | Gauge       | `label_app`=&lt;foo&gt; <br> `namespace`=&lt;namespace&gt; <br> `target_api_version`=&lt;api version&gt; <br> `target_kind`=&lt;target kind&gt; <br> `target_name`=&lt;target name&gt; <br> `verticalpodautoscaler`=&lt;vertical pod autoscaler name&gt;   | EXPERIMENTAL                                                                                                                                                |
 | kube_verticalpodautoscaler_spec_updatepolicy_updatemode                                     | Gauge       | `namespace`=&lt;namespace&gt; <br> `target_api_version`=&lt;api version&gt; <br> `target_kind`=&lt;target kind&gt; <br> `target_name`=&lt;target name&gt; <br> `update_mode`=&lt;foo&gt; <br> `verticalpodautoscaler`=&lt;vertical pod autoscaler name&gt; | EXPERIMENTAL                                                                                                                                                |
 
-## NOTE: Read this before enabling vertical pod autoscalers in your cluster
+## Configuration
 
-Vertical Pod Autoscalers(VPAs) are managed as custom resources. To enable this collector, please ensure that you have its corresponding CRDs installed first. They can be found [here](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/deploy/vpa-beta2-crd.yaml).
+Vertical Pod Autoscalers(VPAs) are managed as custom resources.
+
+To enable the Vertical Pod Autoscaler collector, please:
+
+1. Ensure that the Vertical Pod Autoscaler CRDs are installed in the cluster. The CRDs are [here](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/deploy/vpa-beta2-crd.yaml).
+2. Ensure that `verticalpodautoscalers` is included in list of `Resources` enabled using the flag `--resources` when `kube-state-metrics` is run (see below).
+
+One of the [command line arguments](./docs/cli-arguments.md) for `kube-state-metrics` is `--resources`. If this flag is omitted, a default set of Resources is enabled. This default list does **not** include Vertical Pod Autoscalers.
+
+To enable Vertical Pod Autoscalers, the `kube-state-metrics` flag `--resource` must be included when the binary is run and the list of resources must include `verticalpodautoscalers`.
```

Comment:
```suggestion
To enable Vertical Pod Autoscalers, the `kube-state-metrics` flag `--resource` must be included when the binary is run and the list of resources must include `verticalpodautoscalers`, e.g. `kube-state-metrics --resources=verticalpodautoscalers`.
```

Reply:
Generally, developers will need to include a long list of values for the `--resources` flag including `verticalpodautoscalers`. `--resources=verticalpodautoscalers` on its own would *only* enable `verticalpodautoscalers` and disable all the default resources.